user-guide/Cloud_Platform/Credits/DataMiner_Credits.md

```diff
@@ -0,0 +1,15 @@
+---
+uid: DataMiner_Credits
+---
+
+# DataMiner Credits
+
+DataMiner Credits are a flexible form of currency you can use to subscribe to DataMiner software and [hosting services](https://aka.dataminer.services/usage-based-pricing). See more about [usage-based services](https://aka.dataminer.services/usage-based-services-docs).
+
+Credits will be deducted autmatically when you consume usage-based services. As an example deploying a new DataMiner as a Service (DaaS) system will deduct 1 credit for a period of 2 weeks. <!-- link to DaaS docs -->
```

Comment:
Sounds weird to say 'for a period of 2 weeks', almost as if you get the credit back after this time.

Reply:
Later it might be renewed for another 2 weeks for another credit, not sure how to say it better?

Reply:
'The DataMiner as a Service (DaaS) system will deduct 1 credit every 2 weeks as long as it remains active'

Reply:
How about this?

```suggestion
Credits will be deducted automatically when you consume usage-based services. For example, if you deploy a DataMiner as a Service (DaaS) system for a period of 2 weeks, one credit will be deducted. <!-- link to DaaS docs -->>
```